infrastructure/Program.fs

```diff
@@ -48,122 +52,202 @@ let private getLastCommit () =
-                          "APPINSIGHTS_INSTRUMENTATIONKEY", io applicationsInsight.InstrumentationKey
-                          "WEBSITE_RUN_FROM_PACKAGE", io codeBlobUrl
-                          yield! lastCommitAppSettings ]
-
-                let funcApp =
-                    FunctionApp
-                        (sprintf "azfun-%s-plan" funcName,
-                         FunctionAppArgs
-                             (ResourceGroupName = io resourceGroup.Name,
-                              Name = input (sprintf "azfun-%s-%s" funcName stackName),
-                              AppServicePlanId = io appServicePlan.Id,
-                              StorageConnectionString = io storageAccount.PrimaryConnectionString,
-                              AppSettings = functionAppSettings, SiteConfig = genericSiteConfig, HttpsOnly = input true,
-                              Version = input "~3"))
-
-                (sprintf "%s-app-host-name" funcName, funcApp.DefaultHostname :> obj))
-
-        return dict [ yield! functionHostNames ]
+        let lambdaRole =
+            Iam.Role(
+                "FantomasLambdaRole",
+                Iam.RoleArgs(
+                    AssumeRolePolicy =
+                        input
+                            """{
+                               	"Version": "2012-10-17",
+                               	"Statement": [{
+                               		"Action": "sts:AssumeRole",
+                               		"Principal": {
+                               			"Service": "lambda.amazonaws.com"
+                               		},
+                               		"Effect": "Allow",
+                               		"Sid": ""
+                               	}]
+                               }"""
+                )
+            )
+
+        let _policy =
+            let args =
+                Iam.RolePolicyArgs(
+                    Policy =
+                        input
+                            """{
+	                                "Version": "2012-10-17",
+	                                "Statement": [{
+		                                "Effect": "Allow",
+		                                "Action": [
+			                                "logs:CreateLogGroup",
+			                                "logs:CreateLogStream",
+			                                "logs:PutLogEvents"
+		                                ],
+		                                "Resource": "arn:aws:logs:*:*:*"
+	                                }]
+                                }""",
+                    Role = io lambdaRole.Id
+                )
+
+            Iam.RolePolicy("fantomas-log-policy", args)
+
+        let gateway =
+            let cors =
+                ApiGatewayV2.Inputs.ApiCorsConfigurationArgs(
+                    AllowHeaders = inputList [ input "*" ],
+                    AllowMethods = inputList [ input "*" ],
+                    AllowOrigins = inputList [ input "*" ]
```

Comment:
Do you want to allow all origins are only from https://fsprojects.github.io ?

Reply:
Yeah, I guess I could thighten that a bit indeed.